<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Chui-Wen Chiu</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>檔案(&amp;F)</string>
    </property>
    <addaction name="menuFileNew"/>
    <addaction name="menuFileOpen"/>
    <addaction name="mnuFileSave"/>
    <addaction name="mnuFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="mnuPrintSetting"/>
    <addaction name="mnuPrint"/>
    <addaction name="separator"/>
    <addaction name="mnuExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>編輯(&amp;E)</string>
    </property>
    <addaction name="mnuEditUndo"/>
    <addaction name="separator"/>
    <addaction name="mnuEditCut"/>
    <addaction name="mnuEditCopy"/>
    <addaction name="mnuEditPaste"/>
    <addaction name="mnuEditDel"/>
    <addaction name="separator"/>
    <addaction name="mnuFind"/>
    <addaction name="mnuFileNext"/>
    <addaction name="mnuReplace"/>
    <addaction name="mnuMoveNext"/>
    <addaction name="separator"/>
    <addaction name="mnuSelectAll"/>
    <addaction name="mnuInsertDatetime"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="mnuEditWrap"/>
    <addaction name="mnuViewStyle"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>檢視(&amp;V)</string>
    </property>
    <addaction name="mnuStatusBar"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>說明(&amp;H)</string>
    </property>
    <addaction name="mnuHelp"/>
    <addaction name="separator"/>
    <addaction name="mnuAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuFileNew">
   <property name="text">
    <string>新增(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="menuFileOpen">
   <property name="text">
    <string>開啟舊檔(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="mnuFileSave">
   <property name="text">
    <string>儲存檔案(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mnuFileSaveAs">
   <property name="text">
    <string>另存新檔(&amp;A)...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="mnuPrintSetting">
   <property name="text">
    <string>設定列印格式(&amp;U)...</string>
   </property>
  </action>
  <action name="mnuPrint">
   <property name="text">
    <string>列印(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="mnuExit">
   <property name="text">
    <string>結束(&amp;X)</string>
   </property>
  </action>
  <action name="mnuEditUndo">
   <property name="text">
    <string>復原(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="mnuEditCut">
   <property name="text">
    <string>剪下(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="mnuEditCopy">
   <property name="text">
    <string>複製(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="mnuEditPaste">
   <property name="text">
    <string>貼上(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="mnuEditDel">
   <property name="text">
    <string>刪除(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="mnuFind">
   <property name="text">
    <string>尋找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="mnuFileNext">
   <property name="text">
    <string>找下ㄧ個(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="mnuReplace">
   <property name="text">
    <string>取代(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="mnuMoveNext">
   <property name="text">
    <string>移至(&amp;G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="mnuSelectAll">
   <property name="text">
    <string>全選(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="mnuInsertDatetime">
   <property name="text">
    <string>日期/時間)&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="mnuEditWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自動換行(&amp;W)</string>
   </property>
  </action>
  <action name="mnuViewStyle">
   <property name="text">
    <string>格式(&amp;F)...</string>
   </property>
  </action>
  <action name="mnuStatusBar">
   <property name="text">
    <string>狀態列(&amp;S)</string>
   </property>
  </action>
  <action name="mnuHelp">
   <property name="text">
    <string>說明主題(&amp;H)</string>
   </property>
  </action>
  <action name="mnuAbout">
   <property name="text">
    <string>關於筆記本(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <resources/>
 <connections/>
</ui>
